docs/samples/llmisvc/opt-125m/llm-inference-service-ep-Qwen1.5-MoE-A2.7B-gpu.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: LLMInferenceService
+metadata:
+  name: qwen-1-5-moe-a2-7b
+  annotations:
+    leaderworkerset.sigs.k8s.io/subgroup-exclusive-topology: kubernetes.io/hostname
+spec:
+  model:
+    uri: hf://Qwen/Qwen1.5-MoE-A2.7B
+    name: Qwen/Qwen1.5-MoE-A2.7B
+  parallelism:
+    data: 2
+    dataLocal: 1
+    expert: true
+    tensor: 1
+  router:
+    scheduler: { }
+    route: { }
+    gateway: { }
+  worker:
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: nvidia.com/gpu.product
+                  operator: In
+                  values:
+                    - NVIDIA-H100-80GB-HBM3
```

Comment:
This is the GCP way of tagging, we might need to list all Hopper products or find a generic way to refer to hopper GPUs